packages/ketcher-react/src/script/ui/views/modal/components/Text/Text.tsx

```diff
@@ -136,17 +142,26 @@ const Text = (props: TextProps) => {
     SUBSCRIPT: {
       verticalAlign: 'sub',
       transform: 'scale(0.7)',
-      display: 'inline-block',
       transformOrigin: 'left'
     },
     SUPERSCRIPT: {
       verticalAlign: 'super',
       transform: 'scale(0.7)',
-      display: 'inline-block',
-      transformOrigin: 'left'
+      transformOrigin: 'left',
+      lineHeight: 0
     }
   }
 
+  const refEditor = useRef() as MutableRefObject<Editor>
+  const setFocusInEditor = refEditor.current
+    ? refEditor.current.focus
+    : () => console.log('refEditor not found')
+
+  // set focut on component mount
+  useEffect(() => {
+    setFocusInEditor()
+  })
```

Comment:
As I can see we set focus on any updates. If you want to set focus on mount, you should pass empty array as second argument.